core/Test/Tasty/Core.hs

```diff
@@ -330,10 +346,20 @@ trivialFold :: Monoid b => TreeFold b
 trivialFold = TreeFold
   { foldSingle = \_ _ _ -> mempty
   , foldGroup = \_ _ b -> b
-  , foldResource = \_ _ f -> f $ throwIO NotRunningTests
   , foldAfter = \_ _ _ b -> b
   }
 
+newtype Matched = Matched Bool
```

Comment:
What's wrong with `Any`?

Reply:
Nothing! I wasn't aware of its existence.